### ISSUE TITLE
Is it possible to display an intermediate image of the training process, or to restart training again from the intermediate process?

### ISSUE THREAD

**Issue by gaoming714** [levihsu/OOTDiffusion#192]
Excuse me, is it possible to display an intermediate image of the training process, or to restart training again from the intermediate process? I'd like to be able to pause the pipe so that I can do other things in the middle of the process (like getting a new coat) and thus reduce the overall build time.

**Comment by levihsu** [comment]
Yep. Just save checkpoints and set resume_from_checkpoint in your training script to restart training.
